playground/playground.css

```diff
@@ -316,3 +319,103 @@ footer {
     animation: octocat-wave 560ms ease-in-out;
   }
 }
+
+@media (prefers-color-scheme: dark) {
+  :root {
```

Comment:
might have to add a dark mode switch in playground because there's no other way to switch themes right now. But this is a very good start.

Thanks for the PR ⭐

Reply:
To see the dark theme, you can toggle your systems light / dark setting or toggle your browsers.

https://support.mozilla.org/en-US/kb/change-website-appearance-settings-firefox

https://support.google.com/chrome/answer/9275525

https://support.microsoft.com/en-us/windows/change-colors-in-windows-d26ef4d6-819a-581c-1581-493cfcc005fe

https://support.apple.com/guide/mac-help/use-a-light-or-dark-appearance-mchl52e1c2d2/mac